pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -630,6 +673,31 @@ func setupDefaultVaultOnZfs() error {
 	return nil
 }
 
+// checkAndPublishVaultConfig: Checks if default vault directory exists and if not publishes the vault config
```

Comment:
It would be useful to mention that if /persist/vault (the defaultVault/defaultSecretDataset) does not exist then we assume this is a fresh install and use only the tpm key for the vault.

Reply:
Sure, will update the comment.

Reply:
Updated the comment.